src/js/components/MesosLogView.js

```diff
@@ -281,7 +281,7 @@ class MesosLogView extends mixin(StoreMixin) {
 
     return (
       <pre
-        className="flex-grow flush-bottom"
+        className="flex-grow flush-bottom prettyprint"
```

Comment:
From the looks of the screenshots it still looks like the border is somehow bigger to the left and bottom than top and right